GVRf/Framework/jni/gl/gl_render_buffer.h

```diff
@@ -32,11 +32,12 @@ namespace gvr {
 class GLRenderBuffer {
 public:
     GLRenderBuffer() {
+        deleter_ = getDeleterForThisThread();
```

Comment:
why not try to keep deleter_ as a reference. Then, a lot of reference-to-pointer changes can be saved. Pointers themselves can also more error-prone too.